src/Schema/Values/OwnerType.ts

```diff
@@ -22,6 +22,7 @@ export enum OwnerType {
   collect = "collect",
   collection = "collection",
   collections = "collections",
+  consignmentFlow = "consignmentFlow",
```

Comment:
**nitpick:** We're moving away from copy that says "consignment" on the frontend. Should we start using `sellWithArtsy...` instead of `consignment...`?

Reply:
Good to know! I think I'll keep it as consignment because I think it is less wordy at describing the submission flow